src/AVFoundation/AVAudioSession.cs

```diff
@@ -60,16 +62,16 @@ public NSError SetActive (bool active, AVAudioSessionSetActiveOptions options)
 		}
 #endif
 
-		public NSError SetCategory (NSString theCategory)
+		public NSError? SetCategory (NSString? theCategory)
 		{
 			NSError outError;
 			
-			if (SetCategory (theCategory, out outError))
+			if (theCategory is null || SetCategory (theCategory, out outError))
```

Comment:
Does the SetCategory with the out param take a nullable string? Should it?

Reply:
After talking with you yesterday though, I see why you would say to pass it to the SetCategory method. I guess we could try adding the null arg to the selector and add in the ignore files but the headers do not say it takes a null value @mandel-macaque

Reply:
If I just remove the nullable parameter for 'theCategory' like Rolf suggested below then this is not an issue anymore!